src/screens/StudentGroups.tsx

```diff
@@ -8,72 +8,63 @@ import { FormErrorMessage, FormLabel, FormControl, Input, Button } from '@chakra
 
 import StudentGroup from 'components/StudentGroup'
 
-const Classrooms: React.FC = () => {
-  const { handleSubmit, errors, register, formState, reset } = useForm<FormData>()
+const StudentGroups: React.FC = () => {
+  const { handleSubmit, errors, register, formState } = useForm<FormData>()
 
-  const user = useUser()
+  const { data: user } = useUser()
+
+  const studentGroupsRef = useFirestore().collection('teachers').doc(user.uid).collection('studentGroups')
+  const studentGroupsData = useFirestoreCollectionData(studentGroupsRef)
+  console.log(studentGroupsData)
 
   type FormData = {
     studentGroupName: string
   }
+
+  //Steve is the druel-ist.
   interface IClassroom {
-    docId: string
     studentGroupName: string
     students: []
+    docId: string
```

Comment:
I think it's slightly better to omit this here, i append this field on line 32 with the union operator.  this field only exists on this interface when we explicitly declare that it does, as in line 32.